brmo-loader/src/test/java/nl/b3p/ZakRechtArchiefIntegrationTest.java

```diff
@@ -0,0 +1,183 @@
+ * voor bijvoorbeeld Oracle of
+ * {@code mvn -Dit.test=ZakRechtArchiefIntegrationTest -Dtest.onlyITs=true verify -Ppostgresql > target/postgresql.log}
+ * of
+ * {@code mvn -Dit.test=ZakRechtArchiefIntegrationTest -Dtest.onlyITs=true verify -Pmssql > target/mssql.log}.
+ *
+ * @author meine
+ */
+
+public class ZakRechtArchiefIntegrationTest extends AbstractDatabaseIntegrationTest{
+
+    private static final Log LOG = LogFactory.getLog(ZakRechtArchiefIntegrationTest.class);
+
+    private BrmoFramework brmo;
+
+    private IDatabaseConnection rsgb;
+    private IDatabaseConnection staging;
+
+    private final Lock sequential = new ReentrantLock();
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        BasicDataSource dsStaging = new BasicDataSource();
+        dsStaging.setUrl(params.getProperty("staging.jdbc.url"));
+        dsStaging.setUsername(params.getProperty("staging.user"));
+        dsStaging.setPassword(params.getProperty("staging.passwd"));
+        dsStaging.setAccessToUnderlyingConnectionAllowed(true);
+
+        BasicDataSource dsRsgb = new BasicDataSource();
+        dsRsgb.setUrl(params.getProperty("rsgb.jdbc.url"));
+        dsRsgb.setUsername(params.getProperty("rsgb.user"));
+        dsRsgb.setPassword(params.getProperty("rsgb.passwd"));
+        dsRsgb.setAccessToUnderlyingConnectionAllowed(true);
+
+        rsgb = new DatabaseDataSourceConnection(dsRsgb);
+        staging = new DatabaseDataSourceConnection(dsStaging);
+
+        if (this.isMsSQL) {
+            rsgb.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new MsSqlDataTypeFactory());
+            staging.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new MsSqlDataTypeFactory());
+        } else if (this.isOracle) {
+            rsgb = new DatabaseConnection(OracleConnectionUnwrapper.unwrap(dsRsgb.getConnection()), params.getProperty("rsgb.user").toUpperCase());
+            rsgb.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new Oracle10DataTypeFactory());
+            rsgb.getConfig().setProperty(DatabaseConfig.FEATURE_SKIP_ORACLE_RECYCLEBIN_TABLES, true);
+            staging = new DatabaseConnection(OracleConnectionUnwrapper.unwrap(dsStaging.getConnection()), params.getProperty("staging.user").toUpperCase());
+            staging.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new Oracle10DataTypeFactory());
+            staging.getConfig().setProperty(DatabaseConfig.FEATURE_SKIP_ORACLE_RECYCLEBIN_TABLES, true);
+        } else if (this.isPostgis) {
+            // we hebben alleen nog postgres over
+            rsgb.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new PostgresqlDataTypeFactory());
+            staging.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new PostgresqlDataTypeFactory());
+        }
+CleanUtil.cleanSTAGING(staging);
+        CleanUtil.cleanRSGB_BRK(rsgb, true);
```

Comment:
@mtoonen  Deze moeten naar de `@After cleanup()` methode, alle testcases gaan uit van een schone database, dus de tabellen moeten geleegd na afloop van de test.

waarschijnlijk is dan het falen op de build servers van andere tests ook voorbij, bijv. https://travis-ci.org/B3Partners/brmo/jobs/595598046#L3095
```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.512 s <<< FAILURE! - in nl.b3p.Mantis6235IntegrationTest
[ERROR] testTransformStandberichten(nl.b3p.Mantis6235IntegrationTest)  Time elapsed: 0.511 s  <<< FAILURE!
java.lang.AssertionError: Er zijn geen actuele onroerende zaken expected:<2> but was:<3>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at nl.b3p.Mantis6235IntegrationTest.testTransformStandberichten(Mantis6235IntegrationTest.java:142)
```